### PR TITLE
fix: Upload images using camera to S3

### PR DESCRIPTION
**Pull request by rashmi-sahoo-git**
Updated [Upload Images to and from S3](https://docs.appsmith.com/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images) to address the following:
- Change the title from "Upload Images to and from S3" to "Upload Images Using Camera".
- Remove the "Upload image using Image widget" section.
- Make the topic generic and use S3 as an example.
- Improvise the topic to edit and remove outdated video, field names, and other UI-specific information.
- Add prerequisites section.
- Edit the intro and ensure the style guide is followed.
## Checklist
I have:
- [x] run the content through Grammarly
- [ ] linked to sample apps when relevant
- [ ] added the meta description for each page in the PR
- [x] minimized the callouts and added only when necessary
- [ ] added the `queryString` parameter to the Tabs (if used)
- [ ] masked PII in images. For example, login credentials, account details, and more
- [x] added images only when necessary
- [ ] deleted the images that are no longer used for the updated pages in the PR
- [ ] followed the image file naming convention while renaming or adding new images. (Use lowercase letters, dashes between words, and be as descriptive as possible)
- [ ] used the `<figure/>` tag instead of a markdown representation for images
- [ ] added the `<figcaption/>` tag to add a caption to the image
- [ ] added the `alt` attribute in the `` tag
- [ ] verified and updated the cross-references or created redirect rules for the changed or removed content 
- [x] reviewed and applied the style changes for UI formatting. For example, Bold the UI elements(Buttons on screen) used in the doc.

### CODE REVIEW

File: website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md [hunk 1]
```diff
@@ -2,159 +2,59 @@
...
+   * **File Path**: Path of the location you want to store the file. For example, `images/any`.
+
+     :::info
+     Appsmith automatically creates intermediate folders if they don't exist.
+     :::
+
+   * **File Data Type**: Select **Base64** to upload data from the camera widget.
+   * **Expiry Duration of Signed URL (Minutes)**: Enter an expiration time for the signed URL.
+
+     :::info
```
Comment (jnikhila) [inline website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md:43]: This need not be a callout and can be added to the Expiry duration itself.

File: website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md [hunk 2]
```diff
@@ -2,159 +2,59 @@
...
+
+This page provides steps to upload images using the Camera widget to a datasource and uses the S3 datasource as an example.
+You will learn how to:
+- [Create an S3 datasource and connect your app.](#create-an-s3-datasource)
+- [Upload an image to S3 using the Camera widget.](#upload-an-image-using-the-camera-widget)
+
+## Prerequisites
+- To upload your data to Amazon S3, you must first [create an Amazon S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) in one of the AWS Regions.
+
+## Create an S3 Datasource
```
Comment (jnikhila) [inline website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md:17]: S3 datasource should be a part of the prerequisites, so this section is not needed. You could add that as a bullet after the Amazon S3 bucket to the prerequisites section:
- An Amazon S3 bucket. Link Amazon S3 bucket to official Amazon documentation for creating a bucket.
- A S3 datasource. For more information, see [how to create S3 datasource](Link to S3 reference doc).

File: website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md [hunk 3]
```diff
@@ -2,159 +2,59 @@
...
-You can check this [Guide](/connect-data/how-to-guides/how-to-upload-to-s3)to learn more about Upload/Download Files from S3.
-:::
-
-With Appsmith S3 integration, it is possible to create apps that seamlessly connect with the S3 database and provide additional flexibility for updating and analyzing data.
-
+# Upload Images Using Camera
+Appsmith lets you capture images and videos from your applications and share the data for further use.
+For more information, see [Camera](https://docs.appsmith.com/reference/widgets/camera).
+
+This page provides steps to upload images using the Camera widget to a datasource and uses the S3 datasource as an example.
```
Comment (jnikhila) [inline website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md:9]: The guide will elaborate the steps for uploading images to s3 using the camera widget. Hence, **You will learn how to:** section is not needed.

File: website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md [hunk 4]
```diff
@@ -2,159 +2,59 @@
...
+
+## Upload an image using the Camera widget
+To upload an image or a video to Amazon S3, follow these steps:
+1. Drag and drop the [Camera widget](/reference/widgets/camera) onto the canvas.
+2. In **Queries/JS**, click the **+ Add a new query/JS Object**.
+3. Select your S3 datasource.
+4. In **Commands**, select **Create a new file**.
+5. Configure the following parameters for the new file:
+   * **Bucket Name**: Name of the Amazon S3 bucket.
+   * **File Path**: Path of the location you want to store the file. For example, `images/any`.
```
Comment (jnikhila) [inline website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md:34]: Could you please add an example or rewrite it with respect to the path available in the S3 bucket?

File: website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md [hunk 5]
```diff
@@ -2,159 +2,59 @@
...
+
+     :::info
```
Comment (jnikhila) [inline website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md:36]: Let's add this to the File path bullet itself.

File: website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md [hunk 6]
```diff
@@ -2,159 +2,59 @@
...
+
+     ```
+     {
+	    "data": "{{Camera1.imageDataURL}}"
+     }
+     ```
+
+6. Once all the required parameters have been added, open the property pane of the Camera widget.
+7. In the **OnImageCapture** event, choose your query from the "**execute a query**" option.
+   Query execution occurs when the image is captured and saved.
```
Comment (jnikhila) [inline website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md:58]: Let's rewrite this in an active voice.

**Review by jnikhila — CHANGES_REQUESTED** [review]

File: website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md [hunk 7]
```diff
@@ -2,159 +2,59 @@
...
+   For example:
+
+     ```
+     {
+	    "data": "{{Camera1.imageDataURL}}"
+     }
+     ```
+
+6. Once all the required parameters have been added, open the property pane of the Camera widget.
+7. In the **OnImageCapture** event, choose your query from the "**execute a query**" option.
```
Comment (jnikhila) [inline website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md:57]: As per the style guide, just use bold emphasis for UI elements.

File: website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md (hunk 3 above)
Comment (rashmi-sahoo-git) [reply website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md:9]: Done.

File: website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md (hunk 2 above)
Comment (rashmi-sahoo-git) [reply website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md:17]: Done.

File: website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md (hunk 5 above)
Comment (rashmi-sahoo-git) [reply website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md:36]: Done.

File: website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md (hunk 4 above)
Comment (rashmi-sahoo-git) [reply website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md:34]: Modified the example to "images/camera_uploads". This is the path I have set for my testing.

File: website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md (hunk 1 above)
Comment (rashmi-sahoo-git) [reply website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md:43]: Done.

File: website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md (hunk 7 above)
Comment (rashmi-sahoo-git) [reply website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md:57]: Done.

File: website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md (hunk 6 above)
Comment (rashmi-sahoo-git) [reply website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md:58]: Done.

**Review by danciaclara — APPROVED** [review]

**Review by danciaclara — APPROVED** [review]

**Review by jnikhila — APPROVED** [review]

**Pull request merged**